### PR TITLE
Remove fail2ban from the CI puppet repo.

### PR DESCRIPTION
**Pull request by deanwilson**
After a discussion we no longer want to run Fail2Ban.

https://trello.com/c/7lnrxg1a

### CODE REVIEW

File: Puppetfile [hunk 1]
```diff
@@ -26,8 +26,6 @@ mod 'alphagov/clamav',        :git => 'git://github.com/alphagov/puppet-clamav',
 mod 'alphagov/duplicity',     :git => 'git://github.com/alphagov/puppet-duplicity.git',
                               :ref => 'b9ea2e67ed1bb293fc3e13d4ba20ba3fc0f199f9'
 mod 'alphagov/ext4mount',     :git => 'git://github.com/alphagov/puppet-ext4mount.git'
-mod 'valentinroca/fail2ban',  :git => 'git://github.com/valentinroca/puppet-fail2ban',
```
Comment (mattbostock) [inline Puppetfile]: We'll need to remove the cached tarball also.

**Comment by mattbostock** [comment]
It would be good to say in the commit message why we've decided that Fail2Ban is no longer useful to us.

**Comment by deanwilson** [comment]
Force pushed amended commit with more details/context from the requesting ticket.

**Comment by surminus** [comment]
:+1: good to merge

File: Puppetfile (hunk 1 above)
Comment (mattbostock) [inline Puppetfile]: Please ignore this, we're not vendoring modules in this repo.

**Comment by rjw1** [comment]
probably needs to make the fail2ban package be absent like https://github.com/alphagov/govuk-puppet/pull/4100
although to be honest that can be another pull request or a fabric task in this case.

**Comment by mattbostock** [comment]
@rjw1 I'm going to remove these manually in the interests of getting this deployed.

**Pull request merged**